dev/breeze/tests/test_cache.py

```diff
@@ -62,6 +67,17 @@ def test_read_from_cache_file(param):
     if param_value is None:
         assert None is param_value
     else:
-        allowed, param_list = check_if_values_allowed(param + 's', param_value)
+        allowed, param_list = check_if_values_allowed(param, param_value)
         if allowed:
             assert param_value in param_list
+
+
```

Comment:
@potiuk Could you share your views on this test that I tried? 

In the below code, If i tried `mock_path.assert_called_with(AIRFLOW_SOURCES / ".build"/ param)` it didn't work. Can you explain why it doesn't work? I tried to find myself. But that part is not very clear to me

Reply:
@potiuk Could you share your views on the above question too?